_analyzers/tokenizers/edge-n-gram.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Edge n-gram
+parent: Tokenizers
+nav_order: 40
+---
+
+# Edge n-gram tokenizer
+
+The `edge_ngram` tokenizer generates partial word tokens, or n-grams, starting from the beginning of each word. It splits the text based on specified characters and produces tokens with lengths defined by a minimum and maximum length. This tokenizer is particularly useful for implementing search-as-you-type functionality. 
+
```

Comment:
Line 10: If my rewrite to the second sentence isn't quite right, then let's name what we're referring to: "produces tokens of lengths defined by `min_gram` and `max_gram`."